OpenCM.mk

```diff
@@ -0,0 +1,219 @@
+########################################################################
+#
+# Support for Robotis OpenCM boards
+#
+# http://en.robotis.com/index/product.php?cate_code=131010
+#
+# You must install the OpenCM IDE for this Makefile to work:
+#
+# http://support.robotis.com/en/software/robotis_opencm/robotis_opencm.htm
+#
+# Based on work that is copyright Jeremy Shaw, Sudar, Nicholas Zambetti,
+# David A. Mellis & Hernando Barragan.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# Adapted from Teensy 3.x makefile which was adapted from Arduino 0011
+# Makefile by M J Oldfield
+#
+# Original Arduino adaptation by mellis, eighthave, oli.keller
+#
+########################################################################
+
+ifndef ARDMK_DIR
+    ARDMK_DIR := $(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST)))))
+endif
+
+# include Common.mk now we know where it is
+include $(ARDMK_DIR)/Common.mk
+
+ARDMK_VENDOR         = robotis
+ARDUINO_CORE_PATH    = $(ARDUINO_DIR)/hardware/robotis/cores/robotis
```

Comment:
I followed Teensy's model; ARDUINO_DIR was to be set to the path to the OpenCM_IDE, and all OpenCM boards are robotis.  But I'll change it!